### PR TITLE
Events branch

### PR DESCRIPTION
**Pull request by hui444**
### Create
- Reusable button
- Event-List page

### CODE REVIEW

**Review by cjunxiang — CHANGES_REQUESTED** [review]
Please remove package-lock.json! 
likely generated by npm-install.

File: frontend/src/routes/Home/index.tsx [hunk 1]
```diff
@@ -24,8 +25,15 @@ export default function Home() {
     dispatch(getUpdateMockString())
   }
 
+  const Reddropdownbar = styled(DropDownBar)`
```
[the remark this answers is not in the document]
Comment (hui444) [reply frontend/src/routes/Home/index.tsx]: Removed in commit bb0ac03311a3205bbca964a94e2a1f7a58c63f52

**Comment by cjunxiang** [comment]
Thanks for effort! Looks good to me :-)

**Review by cjunxiang — APPROVED** [review]

**Comment by cjunxiang** [comment]
Try to resolve these before merging! 

After commenting or resolving, LGTM! 
image

**Pull request merged**